### PR TITLE
Add org.gnome.Showtime

### PR DESCRIPTION
**Pull request by kra-mo**
A PyGObject GStreamer video player app.

### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: org.gnome.Showtime.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
+  "id": "org.gnome.Showtime",
```
Comment (hfiguiere) [inline org.gnome.Showtime.json:2]: is this an official GNOME app?

File: org.gnome.Showtime.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  },
+  "command": "showtime",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-pictures/Screenshots:create",
+    "--own-name=org.mpris.MediaPlayer2.org.gnome.Showtime"
```
Comment (hfiguiere) [inline org.gnome.Showtime.json:21]: this name is actually granted by default. (based on the  app-id)

File: org.gnome.Showtime.json (hunk 1 above)
Comment (kra-mo) [reply org.gnome.Showtime.json:2]: Yes, it is in [Incubator](https://gitlab.gnome.org/GNOME/Incubator).

File: org.gnome.Showtime.json (hunk 2 above)
Comment (hfiguiere) [reply org.gnome.Showtime.json]: I don't see what the doc properly update. A lot of app don't use the proper naming which is why they need to own the name.

**Comment by bbhtt** [comment]
bot, build org.gnome.Showtime

**Pull request closed without merging**